pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -0,0 +1,629 @@
+// api.pb.go could be generate by hack/update-generate-runtime.sh
+syntax = 'proto2';
```

Comment:
Let's not use `default` values and `required` fields, unless we really really have to.